### PR TITLE
changes log for MTM-57096

### PR DESCRIPTION
**Pull request by mmed-softwareag**

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1019-4-0-rc-6-restriction-on-role-changes-for-users-managed-by-sso-server.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57096
+version: 1019.4.0-rc.6
+---
+The administrator cannot change the roles of a user managed by the SSO server if they are updated during each login.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-4-0-rc-6-restriction-on-role-changes-for-users-managed-by-sso-server.md:17]:
```suggestion
The administrator can no longer change the roles of a user managed by the SSO server if they are updated during each login.
```

File: content/change-logs/platform-services/ui-c8y-1019-4-0-rc-6-restriction-on-role-changes-for-users-managed-by-sso-server.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Restriction on role changes for users managed by SSO server.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1019-4-0-rc-6-restriction-on-role-changes-for-users-managed-by-sso-server.md:3]:
```suggestion
title: Restriction on role changes for users managed by SSO server
```

File: content/change-logs/platform-services/ui-c8y-1019-4-0-rc-6-restriction-on-role-changes-for-users-managed-by-sso-server.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57096
+version: 1019.4.0-rc.6
+---
+The administrator can no longer change the roles of a user managed by the SSO server if they are updated during each login.
```
[the remark this answers is not in the document]
Comment (kalavathig1) [reply content/change-logs/platform-services/ui-c8y-1019-4-0-rc-6-restriction-on-role-changes-for-users-managed-by-sso-server.md:17]: > I think we need to elaborate this a bit, something like this: The administrator can no longer change the global roles and inventory roles of a user managed by the SSO server if they are updated during each login selected in access mapping of sso configuration

**Review by BeateRixen — APPROVED** [review]

**Review by sarangovindarajan — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@mmed-softwareag @kalavathig1 @sarangovindarajan Is this described in the user documentation?

**Review by kalavathig1 — APPROVED** [review]

**Comment by BeateRixen** [comment]
@kalavathig1 @mmed-softwareag I just merged as the PR was approved but the version is missing.

**Pull request merged**